examples/plot_kernel_ridge_regression.py

```diff
@@ -136,9 +137,9 @@
         test_time.append(time.time() - t0)
 
     plt.plot(sizes, train_time, 'o-', color="r" if name == "SVR" else "g",
-             label="%s (train)" % name)
+             markeredgecolor=(0, 0, 0), label="%s (train)" % name)
```

Comment:
> In this case the dev version is fine, so there is no need to change it.

My personal opinion on this particular plot: dev (no black line around marker) is better than stable (thin black line around marker) and stable is better than this PR  (line around marker is way too wide IMO).